iGOT/client-assets/assets/configurations/site.config.json

```diff
@@ -323,11 +333,47 @@
       ],
       "right": [
         {
+<<<<<<< HEAD
```

Comment:
Resolve merge conflict.

Reply:
@saurTarento That is the only reason for asking for review

Reply:
@saurTarento  lang-assets branch is for CBP, what is the relationship of that with MDO portal?

Reply:
Since you said release-4.0.1-org is the branch for SPV and MDO portal I took a pull of them for prod-lang-assets, branch

Reply:
> @saurTarento lang-assets branch is for CBP, what is the relationship of that with MDO portal?

Because it will become after merging same branch if u use it as comment in assets.